TradeItIosTicketSDK2/TicketRow.swift

```diff
@@ -29,7 +29,7 @@ enum TicketRow {
         case .estimatedCost:
             cellReuseId = .readOnly
         case .quantity, .limitPrice, .stopPrice, .amount:
-            cellReuseId = .numericInput
+            cellReuseId = .numericToggleInput
```

Comment:
Is the old numeric type used anywhere anymore? Can we remove the xib, class and this type?

Reply:
@falnatsheh did this get addressed?